nuget.config

```diff
@@ -2,8 +2,8 @@
 
 <configuration>
   <packageSources>
+    <clear/>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
-    <add key="azure-appservice-staging" value="https://www.myget.org/F/azure-appservice-staging/api/v3/index.json" />
     <add key="azure-appservice" value="https://www.myget.org/F/azure-appservice/api/v3/index.json" />
```

Comment:
this will need to be removed or replaced with an ADO feed as well.

Reply:
ah, I missed there were two of these. Fixing promptly

Reply:
@jviau - fixed in next commit

Reply:
actually, I guess that if we only have `nuget.org`, we don't really need a `nuget.config`....but I think we'll eventually want to add the ADO feed so I'll keep it around

Reply:
You are expected to always have a `nuget.config`, because the `<clear />` element is important.